### DISCUSSION TITLE
Website missing some documentation

### DISCUSSION THREAD

**Discussion by wvh** [prometheus-operator/prometheus-operator#5835] [General]
Hello!

There is some pretty useful documentation in the `user-guides` directory, but most of it is not showing up on the project website. I'm not sure if the website is out-of-date or if there's some other issue, but it might be useful to have those documents there.

Talking about the site at: https://prometheus-operator.dev/

Thanks!

**Comment by slashpai** [comment]
yes. If you look at https://github.com/prometheus-operator/website/blob/main/synchronize.sh#L27-L43 it will show these guides on website
